actnow/petitions/urls.py

```diff
@@ -4,7 +4,7 @@
 from .views import PetitionView, SignatureView, follow_petition
 
 router = routers.DefaultRouter()
-router.register(r"petitions", PetitionView)
+router.register(r"all", PetitionView)
```

Comment:
Can’t this just be empty i.e. `/petitions`

Reply:
The reason it can't be is that `v1/petitions` points to the whole of `include("actnow.petitions.urls")` petitions app. Therefore, we won't be able to do something like `v1/petitions/signatures` since the endpoint expects `signatures` to be an `Id` of a petition.

Reply:
Not sure I get you here

Reply:
I mean, we have a `v1/petitions` endpoint in the main `urls.py` then we register the above routes inside the petitions URLs. so What we get is `v1/petitions/all` and `v1/petitions/petitions`. If we remove the `/all`, the `v1/petitions` will forward every request to the `PetitionView` i.e including the `SignatureView` requests.

Reply:
🤔 

Aren't URLs processed in order written (i.e. top to bottom)? Meaning if we write the routes as:
```python
    router.register(r"signatures", SignatureView)
    router.register(r"", PetitionView)
```
Should work fine.

Reply:
yeah. Forgot about order :facepalm: